frontend/src/main.ts

```diff
@@ -124,7 +225,7 @@ function init(): void {
     errors.set(val.errors);
   });
 
-  router.trigger("page-loaded");
+  initExtensions();
```

Comment:
Loading and running the extension JS code shouldn't block the `page-loaded` event IMHO but should just run asynchronously.

Reply:
I originally did this because I'm using the `page-loaded` event to trigger the `onExtensionPageLoad` extension hook, so if `page-loaded` is triggered before the extensions are loaded, something else would need to call that hook, only once the modules are loaded and if you happen to be navigating directly to the extension page. If that seems ok I can change that around.

Reply:
This doesn't quite work out as it is currently implemented - changes to the ledgerData trigger a re-run of initExtensions which then spuriously triggers "page-loaded". I'll change the logic in my commit